packages/jwt-verifier/README.md

```diff
@@ -131,6 +131,18 @@ const oktaJwtVerifier = new OktaJwtVerifier({
 });
 ```
 
+## Testing
+Setup SPA and Web App in your Okta org and testing environment variables by following [Testing](https://github.com/okta/okta-oidc-js#testing) section in Monorepo's README.
+
+**NOTE:** 
+
+When create SPA application in your Okta org, please make sure all `Implicit` checks have been checked in `General Settings -> Application -> Allowed grant types` section.
```

Comment:
Is this an actual requirement?  If someone is only requesting `access token`, is `id token` grant required?

Reply:
I added `note` here only because it prevent me from running tests successfully without checking `Implicit `. I can remove it if it's not the proper way to declare it. Any suggestions?

Reply:
I'd move that to the `Contributing` section and/or make it clear that they need to be enabled for the tests.  General devs won't be running our tests.

Otherwise this looks like they can't use jwt-verifier without these flags.

Reply:
As it's under `tests` section, just want to confirm is it fine to keep it unchanged?